tests/baselines/reference/jsDeclarationsClassImplementsGenericsSerialization.js

```diff
@@ -68,4 +68,4 @@ export class Encoder<T> implements IEncoder<T> {
      */
     encode(value: T): Uint8Array;
 }
-export type IEncoder<T> = import('./interface').Encoder<T>;
+export type IEncoder<T> = import("./interface").Encoder<T>;

```

Comment:
Since we're doing the recursive visit on the specifier now, the quote-style-conformity change we just made is now also being applied here~